README.md

```diff
@@ -10,6 +17,80 @@ To build this snap locally you need snapcraft. The project must be built as real
 $ sudo snapcraft
 ```
 
+# Testing with spread
+
+## Prerequisites for testing
+
+You need to have the following software installed before you can test with spread
+ - Go (https://golang.org/doc/install or ```sudo snap install go```)
+ - Spread (install from source as per below)
+
+## Installing spread
+
+You can install spread by simply using ```snap install spread```, however this does not allow for the lxd-backend to be used.
+To use the lxd backend you need to install the snap from source, as the LXD profile support has not been upstreamed yet.
```

Comment:
"install the snap" -> "install spread", I guess? (you are providing instructions to build the binary, not the snap)

Reply:
ah good catch, thank you!